SourceKitStressTester/Sources/StressTester/StressTesterTool.swift

```diff
@@ -49,6 +49,12 @@ public struct StressTesterTool: ParsableCommand {
     """, valueName: "page/total"))
   public var page: Page = Page()
 
+  @Option(name: .long, help: ArgumentHelp("""
+    If specified, only execute actions at this offset. \
+    Useful to reproduce a specific failure locally.
+    """))
+  public var offsetFilter: Int?
```

Comment:
Could add a validation that there's only page *or* offset and/or add to the help.